### DISCUSSION TITLE
Removing duplicates of the same comics

### DISCUSSION THREAD

**Discussion by couzin2000** [comictagger/comictagger#575] [Ideas]
I currently am collecting some long-running comic collections (like Iron Man, Hulk, Superman, etc). I've managed to obtain certain files in batches and some issues have been used in the context of "Appearances". So, say Hulk appears in a given Iron Man comic, I'd have the issue in my Hulk folders, but as I am collecting Iron man as well, I also have the exact same issue in my Iron Man folders. 
Wish there was a way to load all files and remove the duplicates.

**Comment by lordwelch** [comment]
Here is some code that I used back with version 3.5.5 I believe, no idea if it still works.
You would need to have ComicTagger installed with pip `pip install comictagger` and then extract this and run `python find_dupes.py path/to/comic/folder` and it would list any files that it found that were duplicates and let you compare them

[find_dupes.zip](https://github.com/comictagger/comictagger/files/13663947/find_dupes.zip)
